README.md

```diff
@@ -188,3 +188,17 @@ var customVocabularySubmission = await client.submitCustomVocabularies(customVoc
 // Get information regarding the Custom Vocabulary submission and its progress
 var customVocabularyInformation = await client.getCustomVocabularyInformation(customVocabularySubmission.id)
 ```
+
+# For Rev.ai NodeSDK Developers
+
+After cloning and installing required npm modules, you should follow these practices when developing:
+
+1. Use the scripts defined in [package.json](https://github.com/revdotcom/revai-node-sdk/tree/develop/package.json) in this manner `npm run [command_name]`:
+    1. `lint` checks that you are not violating any code style standards. This ensures our code's style quality stays high improving readability and reducing room for errors.
+    2. `build` transpiles the typescript into javascript with the options specified in [tsconfig.json](https://github.com/revdotcom/revai-node-sdk/tree/develop/tsconfig.json)
+    3. `unit-test` runs our unit tests which live in the [unit test directory](https://github.com/revdotcom/revai-node-sdk/tree/develop/test/unit).
+    * Note that `integration-test` is currently configured to work with a certain account specified in our continuous integration build environment, as such for now you can check the automated continuous integration checks to pass the integration tests. This will change in the future.
```

Comment:
it uses access_token set by environment variable to run the tests. To run this locally, you should set the value as an env var. 

separately,

What do you mean by `This will change in the future?`

Reply:
So actually, the tests are sort of hard coded to check for a specific job id that belongs to a specific testing account. @Chanc3 might know more. So while you can configure the environment variable locally, the integration tests are currently only meaningful on the CI environment, where the relevant account is configured. This specific configuration which prevents one from running integration tests locally is what I believe `will change in the future`.

Reply:
Remove the comment about this changing in the future.